CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Ansible Changes By Release
 * `stat` and `win_stat` have deprecated `get_md5` and the `md5` return value
   and these options will be removed in Ansible 2.9. `get_md5: no` will still be
   allowed in 2.9 but will finally be removed 2 versions after that.
+* nxos_portchannel module is deprecated in Ansible 2.5. Use nxos_linkagg module instead.
```

Comment:
Please also add to `docs/docsite/rst/porting_guide_2.5.rst`

Reply:
Added.